app/ui/src/app/store/step/step.store.ts

```diff
@@ -62,7 +62,7 @@ export class StepStore {
       action: undefined,
       name: 'Template',
       stepKind: TEMPLATE,
-      description: 'Upload a Mustache template to define a custom step',
+      description: 'Upload or create a Mustache template to define consistent output data.',
```

Comment:
this, too? Not sure if there's a rule for what stuff should be in the dictionary.

Reply:
Yeah, definitely.  I think this will be a bit more tricky because it'd require injecting the i18n service into the store, so again I'm fine leaving it as-is for now, probably worth creating a separate issue specific to this class since there's more than just this one step defined in the step store, all of the steps defined in this class should be using the dictionary.